drivers/scheduler/k8s/specs/nginx/aws/aws-storage-class.yaml

```diff
@@ -0,0 +1,8 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: nginx-sc
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+  fsType: ext4
```

Comment:
this doesn't look right. We might want to use an AWS storage class that supports file (not block) since multiple pods would mount the nginx volume.

Reply:
Well, installing aws file provisioner is quite a big challenge. It is not provided by aws itself and require installing it manually AFAIK. I think we can do that in separate PR if needed, so far I would either sick to RWO or skip apps with RWM pvc.